src/cascadia/TerminalApp/IF7Listener.idl

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+namespace TerminalApp
+{
+    [uuid("339e1a87-5315-4da6-96f0-565549b6472b")]
+    interface IF7Listener
+    {
+        Boolean OnF7Pressed();

```

Comment:
Is there anything else we could use this for in the future that _isn't_ F7? Like, the purpose of this interface is to intercept a keypress. If there's other keypresses that we should intercept, shouldn't we abstract this interface to intercept _any_ specific keypress/action, and rename it as `IInputIntercepter` (or something less specific to F7)?

Reply:
F7 is the _only_ key we need this for, and calling it something F7-specific makes it very clear that this is a point-in-time hack